src/test/java/core/basesyntax/services/imps/ParserServiceImpTest.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.services.imps;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ParserServiceImpTest {
+    private static ParserServiceImp parserService;
```

Comment:
Why to use an implementation class, instead of abstraction? Remember about Dependency Inversion principle.